wger/nutrition/management/commands/import-off-products.py

```diff
@@ -92,25 +141,25 @@ def handle(self, **options):
         self.stdout.write(f' - {self.mode}')
         self.stdout.write('')
 
-        client = MongoClient('mongodb://off:off-wger@127.0.0.1', port=27017)
-        db = client.admin
 
         languages = {l.short_name: l.pk for l in Language.objects.all()}
 
         bulk_update_bucket = []
         counter = Counter()
-
-        for product in db.products.find(
-            {
+        if options['usejsonl']:
+            products=self.products_jsonl(languages=list(languages.keys()),completeness=self.completeness)
```

Comment:
is it just my github or is the alignment off here?
![image](https://github.com/wger-project/wger/assets/4297028/f92a967a-d614-4fc8-873f-123bf5dbfc57)